### PR TITLE
Show webpack icon

### PR DESCRIPTION
**Pull request by James300**
Associated Issue: #2859 

### Summary of Changes

* Added a check for when the item.path is equal to "webpack"
* Renamed `clasName?` to `className?` because I think it was a spelling mistake

### Screenshots/Videos

image

image

image

### CODE REVIEW

File: src/components/PrimaryPanes/SourcesTree.js [hunk 1]
```diff
@@ -166,6 +166,10 @@ class SourcesTree extends Component {
   }
 
   getIcon(item, depth) {
+    if (item.path === "/webpack://") {
+      return <Svg name="webpack" className="webpack-icon" />;
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/SourcesTree.js]: do we need this: `className="webpack-icon"`?

File: src/components/shared/Svg.css [hunk 2]
```diff
@@ -68,3 +68,14 @@ html .arrow.expanded svg {
...
+.webpack svg {
+  width: 15px;
+  height: 15px;
+  margin-right: 5px;
+  vertical-align: bottom;
+}
+
+.theme-dark .webpack {
+  opacity: 0.5;
+}
```
Comment (jasonLaster) [inline src/components/shared/Svg.css]: i think we need a new line here. it's stupid... :)

File: assets/images/Svg.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ const svg = {
 
 type SvgType = {
   name: string,
-  clasName?: string,
+  className?: string,
```
Comment (jasonLaster) [inline assets/images/Svg.js]: nice :)

File: src/components/PrimaryPanes/SourcesTree.js (hunk 1 above)
Comment (James300) [reply src/components/PrimaryPanes/SourcesTree.js]: No we don't, nice catch

**Pull request merged**